### PR TITLE
GIT events promo work

### PR DESCRIPTION
**Pull request by EllieNodder**
### Trello card

https://trello.com/c/i9mZKrgK/3753-change-promo-modules-to-events

### Context

Now that Get Into Teaching events are starting again, we should look at promoting these around the website.

This includes:

- adding GIT events promos to relevant pages
- making sure the about GIT events page is optimised properly

### CODE REVIEW

File: app/views/teaching_events/about_git_events.html.erb [hunk 1]
```diff
@@ -60,7 +60,7 @@
       <section class="container no-margin bottom-padding">
         <div>
           <h2 class="heading-s">Can't find what you're looking for?</h2>
-          <p><%= link_to("View all events", events_path, class: "link--bold") %></p>
+          <p>You can also find <%= link_to("events in your area hosted by teacher training providers", events_path, class: "link--bold") %>. Or, take a look at our <%= link_to("online events", events_path(add_online_events(params))) %>.</p>
```
Comment (EllieNodder) [inline app/views/teaching_events/about_git_events.html.erb:63]: @ethax-ross would you be able to fix these links for me pls? The 'events in your area hosted by teacher training providers' link should take them to training provider events (if poss), and the 'online events' to the online Q&A events.

File: app/views/content/train-to-be-a-teacher/promos/_events-promo-apply.html.erb [hunk 2]
```diff
@@ -0,0 +1,13 @@
+
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get Into Teaching events") do %>
+  <% end %>
+  <% promo.right_section(
+        caption: "Online and in-person events",
+        heading: "Get tips on your application") do %>
+      <p>Talk to teacher training providers, expert advisers and current and former teachers to get help writing your teacher training application.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_events-promo-apply.html.erb:9]:
```suggestion
      <p>Talk to teacher training providers, expert advisers and teachers to get help writing your teacher training application.</p>
```
Do we need to say 'current and former' - does this add anything?

File: app/views/content/train-to-be-a-teacher/promos/_events-promo-degree.html.erb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get Into Teaching events") do %>
+  <% end %>
+  <% promo.right_section(
+        caption: "Online and in-person events",
+        heading: "Find out more about training to teach") do %>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_events-promo-degree.html.erb:8]: Is there any way we can shorten this to get in on one line on mobile/desktop?

File: app/views/content/funding-and-support/promos/_events-promo-disabled.html.erb [hunk 4]
```diff
@@ -0,0 +1,13 @@
+
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get Into Teaching events") do %>
+  <% end %>
+  <% promo.right_section(
+        caption: "Online and in-person events",
+        heading: "Find out more about the support available") do %>
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/promos/_events-promo-disabled.html.erb:8]: Can we shorten this to one line? doesn't really feel like a heading?

File: app/views/content/funding-and-support/promos/_events-promo-parent.html.erb [hunk 5]
```diff
@@ -0,0 +1,13 @@
+
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get Into Teaching events") do %>
+  <% end %>
+  <% promo.right_section(
+        caption: "Online and in-person events",
+        heading: "Find out more about the support available") do %>
```
Comment (gemmadallmandfe) [inline app/views/content/funding-and-support/promos/_events-promo-parent.html.erb:8]: Can we shorten this to one line? doesn't really feel like a heading?

File: app/views/content/train-to-be-a-teacher/promos/_events-promo-ske.html.erb [hunk 6]
```diff
@@ -0,0 +1,13 @@
+
+<%= render(CallsToAction::Promo::PromoComponent.new) do |promo| %>
+  <% promo.left_section(
+      heading: "Get Into Teaching events") do %>
+  <% end %>
+  <% promo.right_section(
+        caption: "Online and in-person events",
+        heading: "Get your questions answered") do %>
+      <p>Talk to advisers and teacher training providers at a Get Into Teaching event to find out if you need to do a subject knowledge enhancement (SKE) course.</p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_events-promo-ske.html.erb:9]:
```suggestion
      <p>Talk to advisers and teacher training providers at a Get Into Teaching event to find out if you need to do a subject knowledge enhancement (SKE) course.</p>
```

I think this is slightly misleading - it is up to the providers you apply to as to whether you need to do SKE as a condition of offer - TTAs and the providers at the events won't be able to say definitively.

Better to say 'find out more about....' or similar?

File: app/views/teaching_events/about_git_events.html.erb (hunk 1 above)
Comment (gemmadallmandfe) [reply app/views/teaching_events/about_git_events.html.erb]: these links are fine @EllieNodder - there are just no events of either type in the dev CRM

I've tested the links in prod and they are fine

Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb]: Two things:

* the provider link is in bold and it shouldn't be
* I think we should put the DfE online Q&A link first eg

You can also take a look at our online Q&A events or provider events in your area.

**Comment by gemmadallmandfe** [comment]
> 

makes sense @EllieNodder - let's do that in a separate PR afterwards!

**Comment by gemmadallmandfe** [comment]
> > One more comment @EllieNodder - should we change the 'Here to help' image on the promo module to the event image we are using more widely including on the home page hero?
> 
> @gemmadallmandfe Think that makes sense - though the image has been added in [this PR](https://github.com/DFE-Digital/get-into-teaching-app/pull/2892/files#diff-eb94588c7d14f61896eb3893e99dd6ead5c6f39c8ea68b58ba98c29557538e27) so will either have to merge these two together or create a new PR to switch out the image once that one has gone live.

makes sense @EllieNodder - let's do that in a separate PR afterwards!

File: app/views/teaching_events/about_git_events.html.erb [hunk 7]
```diff
@@ -16,7 +16,7 @@
       <p>These relaxed and informative events give you the opportunity to get expert advice and talk to teachers and local training providers.</p>
     </section>
     <section>
-      <h2 class="heading-m">At the event</h2>
+      <h2 class="heading-m">At the Get Into Teaching event</h2>
```
Comment (gemmadallmandfe) [inline app/views/teaching_events/about_git_events.html.erb:19]:
```suggestion
      <h2 class="heading-m">At our Get Into Teaching events</h2>
```
bit friendlier? marries up more closely with ad straplines?

File: app/views/content/train-to-be-a-teacher/promos/_adviser-promo-ske.html.erb [hunk 8]
```diff
@@ -4,7 +4,7 @@
   <% end %>
 
   <% promo.right_section(heading: "Get free one-to-one support") do %>
-      <p>Your adviser can help you understand how to do a subject knowledge enhancement course and support you with your teacher training application.</p>
+      <p>Your adviser can help you understand how to do a subject knowledge enhancement course and support you with your teacher training application. </p>
```
Comment (gemmadallmandfe) [inline app/views/content/train-to-be-a-teacher/promos/_adviser-promo-ske.html.erb:7]: did you mean to change this @EllieNodder ?

**Comment by gemmadallmandfe** [comment]
Looks good @EllieNodder 

Two minor comments above

plus

* should we change this on Scholarships and bursaries as well (see comment on Trello card from Events team) - can be done in a separate PR if easier
* image to change in a separate PR

Will merge this tomorrow as soon as you've had a chance to look at the comments

File: app/views/content/train-to-be-a-teacher/promos/_adviser-promo-ske.html.erb (hunk 8 above)
Comment (EllieNodder) [reply app/views/content/train-to-be-a-teacher/promos/_adviser-promo-ske.html.erb:7]: GitHub was throwing a tantrum and not working so just added this change in to kick it back to life - will take it back out

**Comment by EllieNodder** [comment]
@gemmadallmandfe have added a promo to the scholarships page

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me
